utilities/infra.py

```diff
@@ -112,8 +112,10 @@
 
 
 def label_project(name, label, admin_client):
-    ns = Namespace(client=admin_client, name=name)
-    ResourceEditor({ns: {"metadata": {"labels": label}}}).update()
+    ns = Namespace(client=admin_client, name=name, ensure_exists=True)
+    ns.wait_for_status(status=Namespace.Status.ACTIVE, timeout=TIMEOUT_2MIN)
```

Comment:
Why do you want to wait for `status.active`?
There might be a test that labels a non active namespace, I don't think it should be enforced.